src/content/en/fundamentals/performance/critical-rendering-path/measure-crp.md

```diff
@@ -61,3 +61,26 @@ The above example may seem a little daunting on first sight, but in reality it i
 
 All said and done, we now have some specific milestones to track and a simple function to output these measurements. Note that instead of printing these metrics on the page you can also modify the code to send these metrics to an analytics server ([Google Analytics does this automatically](https://support.google.com/analytics/answer/1205784)), which is a great way to keep tabs on performance of your pages and identify candidate pages that can benefit from some optimization work.
 
+## Auditing a page with Lighthouse {: #lighthouse }
+
+Lighthouse is a web app auditing tool that runs a series of tests against a
+given page, and then displays the page's results in a consolidated report. You
+can run Lighthouse as a Chrome Extension or NPM module (which is
+useful for integrating Lighthouse with continuous integration systems).
+
+Lighthouse has a suite of audits for measuring a page's CRP. When you
+run Lighthouse as a Chrome Extension, your page's CRP results look like the
+screenshot below.
+
+![Lighthouse's CRP audits](images/lighthouse-crp.png)
+
+See [Auditing Web Apps With Lighthouse](/web/tools/lighthouse) to get started.
+
+## What about DevTools? {: #devtools }
+
+Although these docs sometimes use the Chrome DevTools Network panel to
+illustrate CRP concepts, DevTools is currently not well-suited for CRP
+measurements. The problem is that, as of October 2016, there is no
+filter for identifying critical resources. In other words, on a page
+that fetches tens or hundreds of resources, there is no way to filter
+out the non-critical resources.
```

Comment:
Could we simplify this / avoid dates, etc?

"DevTools does not have a built-in mechanism to isolate the critical resources. To help identify such resources, try running a Lighthouse audit, or review guidance in [link to docs] to identify them manually."